Docs/User Guide.md

```diff
@@ -0,0 +1,88 @@
+## Overview
+
+Many HIL test systems require hardware fault injection to simulate faults between the DUT and the rest of the system. The **Routing and Faulting Custom Device** injects faults using routing modules. The **SLSC Switch Custom Device** provides hardware support for SLSC Switch routing modules.
+
+The routing module acts as a gate between the DUT and the I/O. A common configuration in a fault injection application is a "fault bus topology" where endpoints are shorted to one or more buses.
+
+| States | Description | Diagram |
+|---|---|---|
+| Passthrough | The DUT and the I/O are connected without fault. This is the default mode of operation. | ![Passthrough](https://ni.scene7.com/is/image/ni/wraiiira5591242785034529104?scl=1) |
+| Open Circuit | The DUT and the I/O are not connected simulating an open circuit. | ![Open Circuit](https://ni.scene7.com/is/image/ni/lrknfeim3638940521012328059?scl=1)|
+| Short to Ground | The DUT and the system ground are connected simulating a short to ground. | ![Short to Ground](https://ni.scene7.com/is/image/ni/yijyimpy803861224648293470?scl=1) |
+| Short to Power | The DUT and the system power are connected simulating a short to power. | ![Short to Power](https://ni.scene7.com/is/image/ni/yijyimpy803861224648293470?scl=1) |
+| Pin-to-Pin Short | Two or more DUT pins are connected together simulating a pin-to-pin short. | ![Pin-to-pin short](https://ni.scene7.com/is/image/ni/bbkmcugz4974244261650505563?scl=1) |
+
+## SLSC Switch Custom Device
+
+The **SLSC Switch Custom Device** is a VeriStand add-on that supports SLSC routing modules. The following modules are supported:
+* [SLSC-12251](https://www.ni.com/en-us/support/model.slsc-12251.html)
+* [SLSC-12252](https://www.ni.com/en-us/support/model.slsc-12252.html)
+* [SET-2010](https://www.smart-e-tech.de/en/products/signal-conditioning-with-slsc/set-slsc-cards/set-2010-routing-card/)
+
+SLSC Switch routing modules are displayed in the VeriStand System Explorer under _Targets >> Controller >> Hardware >> SLSC >> SLSC Chassis >> Modules >> Slot (n)_.
+
+![SLSC Switch System Explorer](SLSC%20Switch%20System%20Explorer.png)
+
+The **SLSC Switch Custom Device** enumerates all available endpoints for a routing module. Set the configuration for each endpoint to inform the SLSC Switch driver which endpoints are valid to connect.
+
+| Configuration | Description |
+|---|---|
+| Load | A load endpoint is unrestricted and can be connected to other load and source endpoints. This is the default configuration for an endpoint. |
+| Source | A source endpoint is restricted and cannot be connected to another source endpoint. |
+| Configuration | A configuration endpoint is reserved for internal routing and cannot be used directly. |
+
+_NOTE: Configure the SLSC Switch Custom Device before configuring the Routing and Faulting Custom Device._
+
```

Comment:
probably worth referencing this white paper in SLSC Switch section or References section: https://www.ni.com/en-us/innovations/white-papers/19/using-the-slsc-12251-2-fault-insertion-and-current-sensing-unit.html

After the note. we could add:
"Here's [some more background](https://www.ni.com/en-us/innovations/white-papers/19/using-the-slsc-12251-2-fault-insertion-and-current-sensing-unit.html) on how to use an SLSC module for fault insertion."